### PR TITLE
feat: ability to set the environment variables with envFrom in k6 runner/starter.

### PR DESCRIPTION
**Pull request by mugioka**
Closes https://github.com/grafana/k6-operator/issues/122.

### CODE REVIEW

File: config/crd/bases/k6.io_k6s.yaml [hunk 1]
```diff
@@ -988,13 +988,35 @@ spec:
                     type: object
                   serviceAccountName:
                     type: string
+                  envFrom:
```
Comment (mugioka) [inline config/crd/bases/k6.io_k6s.yaml:991]: definition is [here](https://github.com/kubernetes/kubernetes/blob/7e2c02d4c86d73051b6d4f47298d35a7ffb42b6a/api/openapi-spec/v3/apis__apps__v1_openapi.json#L1681)

Comment (yorugac) [reply config/crd/bases/k6.io_k6s.yaml:991]: Could you please add this as a comment?

**Review by yorugac — CHANGES_REQUESTED** [review]
@mugioka Thanks for the PR! It LGTM :+1: I'll run a couple of tests but should be good to merge. Please see there is a small request change.

File: config/crd/bases/k6.io_k6s.yaml (hunk 1 above)
Comment (mugioka) [reply config/crd/bases/k6.io_k6s.yaml:992]: I fixed it with https://github.com/grafana/k6-operator/pull/129/commits/1559ba0f183fbc9c89fadd736f1dd17dc78bd354.

**Pull request merged**